Ports/ja2/package.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=Jagged-Alliance-2
+version=0.15.x
+depends=("SDL2")
+workdir=ja2-stracciatella-0.15.x
+files="https://github.com/ja2-stracciatella/ja2-stracciatella/archive/refs/heads/${version}.zip ja2-stracciatella-0.15.x.zip 178375de4859d16a76276c781455bf48d3fa862841387c8aa6cfa4162f4f0ca4"
```

Comment:
Please hardcode the version in as few places as possible:

```suggestion
workdir="ja2-stracciatella-${version}"
files="https://github.com/ja2-stracciatella/ja2-stracciatella/archive/refs/heads/${version}.zip ja2-stracciatella-${version}.zip 178375de4859d16a76276c781455bf48d3fa862841387c8aa6cfa4162f4f0ca4"
```